app/scripts/templates/connect_another_device.mustache

```diff
@@ -0,0 +1,52 @@
+<div id="main-content" class="card">
+  <header class="hidden"><h1>{{#t}}Connect another device{{/t}}</h1></header>
+  <div class="success visible">{{#t}}Email confirmed{{/t}}</div>
+  <section>
+    <div class="graphic graphic-connect-another-device">{{#t}}Success{{/t}}</div>
+
+    {{#canSignIn}}
+      <p>
+        {{#t}}Complete set-up by signing in to this Firefox.{{/t}}
+      </p>
+
+      <form novalidate>
+        <div class="button-row">
+          <a id="signin" class="button" href="{{{ escapedSignInUrl }}}">{{#unsafeTranslate}}Sign in &rarr;{{/unsafeTranslate}}</a>
```

Comment:
Maybe we should log a click on these external links so we know where users are exiting. That info doesn't seem to be logged anywhere.